package.json

```diff
@@ -17,19 +17,19 @@
     "license": "ISC",
     "homepage": "",
     "devDependencies": {
-        "clean-webpack-plugin": "^3.0.0",
-        "copy-webpack-plugin": "^7.0.0",
-        "css-loader": "^5.1.0",
-        "eslint": "^7.27.0",
-        "eslint-config-prettier": "^8.3.0",
-        "eslint-plugin-import": "^2.23.3",
-        "eslint-plugin-prettier": "^3.4.0",
-        "html-webpack-plugin": "^5.2.0",
-        "prettier": "2.2.1",
-        "style-loader": "^2.0.0",
-        "webpack": "^5.37.1",
-        "webpack-cli": "^4.5.0",
-        "webpack-dev-server": "^3.11.2",
-        "webpack-merge": "^5.7.3"
+        "clean-webpack-plugin": "^4.0.0",
+        "copy-webpack-plugin": "^12.0.2",
+        "css-loader": "^6.10.0",
+        "eslint": "^8.56.0",
+        "eslint-config-prettier": "^9.1.0",
+        "eslint-plugin-import": "^2.29.1",
+        "eslint-plugin-prettier": "^5.1.3",
```

Comment:
I think we should get rid of `eslint-plugin-prettier`, it's no longer recommended on official Prettier website

Reply:
It's personal preference thing even if prettier not recommended it, so I believe there is always be both sides